python/mxnet/ndarray/ndarray.py

```diff
@@ -105,6 +107,8 @@
 _NDARRAY_BASIC_INDEXING = 0
 _NDARRAY_ADVANCED_INDEXING = 1
 
+# Caching whether MXNet was built with INT64 support or not
+_INT64_TENSOR_SIZE_ENABLED = Features().is_enabled('INT64_TENSOR_SIZE')
```

Comment:
This is still not efficient as Features() is still called everytime this variable is evaluated. There is no const variable in Python as in C++